### PR TITLE
Fix a number of trivial typos

### PR DESCRIPTION
**Pull request by kyuyeonpooh**
## Why are these changes needed?

There were some bunch of trivial typos and were fixed.
For migration-example.yaml, num-cpus should be stated as string as mentioned in [here](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/config.html#num-cpus).

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [x] This PR is not tested :(

### CODE REVIEW

**Review by architkulkarni — APPROVED** [review]
Thanks for catching these!

**Comment by kyuyeonpooh** [comment]
@architkulkarni Is there anything needs to be improved for this pr to be merged?

**Review by DmitriGekhtman — APPROVED** [review]

**Comment by DmitriGekhtman** [comment]
Looks like we need more code owner approvals.

**Review by pcmoritz — APPROVED** [review]

**Pull request merged**
